x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -1238,6 +1243,19 @@ private static SystemIndexDescriptor getSecurityMainIndexDescriptor() {
              .setIndexFormat(INTERNAL_MAIN_INDEX_FORMAT)
              .setVersionMetaKey("security-version")
              .setOrigin(SECURITY_ORIGIN)
+             .setPriorSystemIndexDescriptors(org.elasticsearch.common.collect.List.of(
+                 SystemIndexDescriptor.builder()
+                     .setIndexPattern(".security-[0-9]+")
+                     .setPrimaryIndex(RestrictedIndicesNames.INTERNAL_SECURITY_MAIN_INDEX_7)
+                     .setDescription("Contains Security configuration")
+                     .setMappings(getIndexMappings(Version.V_7_13_0.getPreviousVersion()))
```

Comment:
I was surprised that we don't already have the `Version#getPreviousVersion` method. Maybe my need here is overthinking it?

Reply:
I don't think you actually want this here.
You don't know what the index mappings for "version previous to 7.13.0" are, you only know what they are for an explicit concrete version. I think you want to say this is the "7.12.0" version because you don't know what the "7.12.2" is going to look like.

Reply:
I think you are right. Will replace this with `Version.V_7_12_0` and remove relevant changes for `Version` as well.